Alignment/OfflineValidation/python/TkAlAllInOneTool/configTemplates.py

```diff
@@ -17,6 +17,16 @@
 
 ######################################################################
 ######################################################################
+loadGlobalTagTemplate="""
+#Global tag
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+from Configuration.AlCa.GlobalTag import GlobalTag
+process.GlobalTag = GlobalTag(process.GlobalTag,".oO[GlobalTag]Oo.")
```

Comment:
@hroskes to answer your second question: currently, we are not really doing it the recommended way in the all-in-one tool because we are creating an ESSource for every condition and then add an ESPrefer. What Marco meant was to use directly the GlobalTag ESSource and append conditions to the `toGet` list of it as is done [here](https://github.com/cms-sw/cmssw/blob/CMSSW_8_1_X/Alignment/MillePedeAlignmentAlgorithm/python/alignmentsetup/SetCondition.py) for instance.